infomaniak-build-tools/windows/build-drive.ps1

```diff
@@ -442,9 +442,18 @@ if (Test-Path -Path $installerPath)
 else
 {
 	Write-Host ("$installerPath not found. Unable to sign final installer.") -f Red
-	exit $LASTEXITCODE
+	exit 1
 }
 
+#################################################################################################
```

Comment:
Then ending `#` are not aligned.

Reply:
Changed tabs to spaces to prevent this display issue